functional-samples/sample.dnr-rule-manager/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "no-cookies Rule Manager",
+  "version": "0.1",
+  "manifest_version": 3,
+  "description": "Uses the chrome.declarativeNetRequest API to remove the \"Cookie\" header from requests matching custom url filters.",
```

Comment:
Given this is used by the samples page, I think we should say more about how this differs from the other sample (e.g gives you a way to manage rules)?